hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/mgmt/documentation/RolloutResourceDocumentationTest.java

```diff
@@ -130,6 +130,8 @@ private Snippet getRolloutResponseFields(final boolean isArray, final boolean wi
                 fieldWithPath(arrayPrefix + "distributionSetId").description(MgmtApiModelProperties.ROLLOUT_DS_ID));
         allFieldDescriptor.add(fieldWithPath(arrayPrefix + "status").description(MgmtApiModelProperties.ROLLOUT_STATUS)
                 .attributes(key("value").value("['creating','ready','paused','running','finished']")));
+        allFieldDescriptor.add(fieldWithPath(arrayPrefix + "type").description(MgmtApiModelProperties.ROLLOUT_TYPE)
```

Comment:
What about [TargetResourceDocumentationTest](https://github.com/eclipse/hawkbit/blob/master/hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/mgmt/documentation/TargetResourceDocumentationTest.java#L522)? Is there no `downloadonly` possible?

Reply:
... same for [DistributionSetsDocumentationTest](https://github.com/eclipse/hawkbit/blob/master/hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/mgmt/documentation/DistributionSetsDocumentationTest.java#L394)

Reply:
good catch, will adapt the tests